src/main/proto/ga4gh/bio_metadata.proto

```diff
@@ -48,8 +48,15 @@ message Individual {
   // Recommended: PATO http://purl.obolibrary.org/obo/PATO_0020001; PATO_0020002
   OntologyTerm sex = 9;
 
+  // External identifiers representing this individual.
+  // TODO: Define if those are a different representation of the same
+  // record, or also can refer to instances containing this record, e.g.:
+  //  - publication identifiers with this record
+  //  - experimental series, datasets containing this record
+  repeated ExternalIdentifier external_identifiers = 10;
+
   // A map of additional information regarding the Individual.
-  Attributes attributes = 10;
+  Attributes attributes = 11;
```

Comment:
We cannot renumber existing fields. New fields should get the next higher unused number. The numbering will appear out of order but renumbering a field breaks compatibility with earlier implementations of the API.